### DISCUSSION TITLE
Nested config folder

### DISCUSSION THREAD

**Discussion by schergui-thefork** [kedro-org/kedro#1709] [Idea]
I'm trying to organize my pipelines within folders by main themes. For e.g, I'd like to have something like:

-processing
--processing_pipeline_1
--processing_pipeline_2
-modelling
--modelling_pipeline_1
--modelling_pipeline_2
--modelling_pipeline_3

So within the conf folder, I'd like to reproduce the same structure.

-conf
--base
---processing
----processing_pipeline_1
----processing_pipeline_2
---modelling
----modelling_pipeline_1
----modelling_pipeline_2
----modelling_pipeline_3

 Is there a way to accomplish this without having to specify a partial path when calling kedro run to run a pipeline?
I don't want to do this `kedro run --pipeline processing_pipeline_2 --env base/processing` and I simply would like to say `kedro run --pipeline processing_pipeline_2 --env base`

**Comment by astrojuanlu** [comment]
Closing this discussion in favour of https://github.com/kedro-org/kedro/discussions/3871
